services/elasticsearch-docker.json

```diff
@@ -22,22 +22,21 @@
       "install": {
         "type": "docker",
         "fields": {
-          "image_name": "elasticsearch",
-          "publish_ports": "9200,9300"
+          "image_name": "elasticsearch"
         }
       },
       "start": {
         "type": "docker",
         "fields": {
           "image_name": "elasticsearch",
-          "publish_ports": "9200,9300"
+          "publish_ports": "9200,9300",
+          "command": "elasticsearch -Des.network.host=0.0.0.0"
```

Comment:
No, command name to send to entrypoint script as an argument, overriding the elasticsearch image's CMD (`elasticsearch`) with our own (`elasticsearch -Des.network.host=0.0.0.0`)... everything in `command` is considered an argument to the container. If the container has an entrypoint, like this one, then it will be an argument to that script. If the container does not have an entrypoint, then it is a command to be executed.